pkg/agent/openflow/client.go

```diff
@@ -538,11 +538,27 @@ func (c *client) UninstallEndpointFlows(protocol binding.Protocol, endpoint prox
 	return c.deleteFlows(c.serviceFlowCache, cacheKey)
 }
 
-func (c *client) InstallServiceFlows(groupID binding.GroupIDType, svcIP net.IP, svcPort uint16, protocol binding.Protocol, affinityTimeout uint16) error {
+func (c *client) InstallServiceFlows(groupID binding.GroupIDType, svcIP net.IP, svcPort uint16, protocol binding.Protocol, affinityTimeout uint16, isNodePortLocal bool) error {
```

Comment:
Humm..
NodePortLocal might mislead people to think about the Antrea NodePortLocal feature? How about nodeLocalExternal?